app/services/exchange_rates/monthly_exchange_rates_service.rb

```diff
@@ -19,18 +19,21 @@ def call
         rates,
         date,
         :monthly_csv,
+        sample_date,
       ).call
 
       ExchangeRates::UploadFileService.new(
         rates,
         date,
         :monthly_xml,
+        sample_date,
       ).call
 
       ExchangeRates::UploadFileService.new(
         rates,
         date,
```

Comment:
I wondered if we can have better naming for `date` this is the same as the sample date right?

Reply:
Think its just a bit confusing where we have `selected_date`, `date` and `sample_date` unless im not understanding